packages/client/e2e/_utils/page.ts

```diff
@@ -92,6 +92,13 @@ export async function setupAndGetUserPage({
     };
 }
 
+export async function initPage(page: Page): Promise<void> {
```

Comment:
I think we should also be using this function in the above function setupAndGetUserPage

Reply:
You're right!
I missed it.